pages/collection/[chain]/[contract].tsx

```diff
@@ -484,7 +481,7 @@ export const getStaticProps: GetStaticProps<{
 
   return {
     props: { ssr: { collection, tokens, attributes }, id },
-    revalidate: 20,
+    revalidate: 60,
```

Comment:
Maybe we can set this to 30, as per Peter's request of keeping it as realtime as possible.